### ISSUE TITLE
Enter key causes comment posting when using IME

### ISSUE THREAD

**Issue by korikisulda** [retrospring/bugs#82]
I sometimes type in Japanese. Unfortunately, finishing composing by pressing the enter key causes the comment to post. It means that it's impossible for me to write any kind of sentence in Japanese as a question comment.

The comment I discovered this on would have started with 新世紀エヴァンゲリオン. In order to input this, I would type shinseiki(space)(enter)evangerion(space)(enter).

**Comment by pixeldesu** [comment]
We will look into a way to change this, thanks for reporting!

**Comment by korikisulda** [comment]
Thanks :D

**Comment by yukimono** [comment]
Still an ongoing [firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=354358) engine bug, Webkit stops capturing enter events while using IME since it doesn't actually *input* the enter key, but firefox doesn't do this.

One way to fix this is to (on firefox) is to see if the input field gained more than 1 character

i.e.

```javascript
input.staticValue = 0;
input.on("keydown", function(event) { input.staticValue = this.value.length; }).
      on("keyup", function(event) { 
        if(event.which == 13 && input.statucValue.length - this.value.length > 1) {
          send(this);
       }
      });
```

**Comment by yukimono** [comment]
Firefox also apparently sends the 229 keycode (å) when IME starts 
so you could do something like

```javascript
input.on("keyup", function(event) {
  $t = $(this);
  if(event.which == 229) {
    $t.data("lock", 1);
  } else if(event.which == 13) {
    if($.data("lock")) {
      $t.data("lock", 0);
    } else {
      send(this);
    }
  }
});
```

**Comment by yukimono** [comment]
tl;dr Firefox's JavaScript engine doesn't know what to do with IME, for a very very very long time now.
